3회차/윤효근/1_모음이보이지않는사람.py

```diff
@@ -1,3 +1,19 @@
 import sys
-
 sys.stdin = open("_모음이보이지않는사람.txt")
+info = ['a','e','i','o','u']
+T = int(input())
+
+for test_case in range(1,T+1):
+    s = input()
+    count=0
+    stack = []
+    for i in s:
+        for j in range(len(info)):
+            if i == info[j]:
+                count += 1
```

Comment:
좋은 풀이네요! 👍 
`for`문 대신 `in`을 사용해도 괜찮을 것 같습니다 😄 
```suggestion
        if i in info:
            count+= 1
```